mysql-paas-replication/nested/mysqlReplica.json

```diff
@@ -0,0 +1,53 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "sku": {
+            "type":  "object"
+        },
+        "serverName": {
+            "type": "string"
+        },
+        "location": {
+            "type": "string"
+        },
+        "numberOfReplicas": {
+            "type": "int"
+        },
+        "backupRetentionDays": {
+            "type": "int"
+        },
+        "geoRedundantBackup": {
+            "type": "string"
+        },
+        "sourceServerId": {
+            "type": "string"
+        }
+    },
+    "variables": {},
+    "resources": [
+        {
+            "copy": {
+                "name": "replicasLoop",
+                "count": "[if(equals(parameters('numberOfReplicas'),0),add(parameters('numberOfReplicas'),1),parameters('numberOfReplicas'))]",
+                "mode": "serial"

```

Comment:
Unfortunately yes it does. The first execution of this nested deployment will cause a change to the original MySQL resource in the Azure's back-end. Essentially it will enable binary logging and restart the MySQL server.